src/printer/kitty.rs

```diff
@@ -179,7 +265,7 @@ fn print_remote(
     // write the first chunk, which describes the image
     write!(
         stdout,
-        "\x1b_Gf=32,a=T,t=d,s={},v={},c={},r={},m=1;{}\x1b\\",
+        "\x1b_Gf=32,a=T,t=d,s={},v={},c={},r={},i=10,m=1;{}\x1b\\",
```

Comment:
> Additionally, while i was at it and noticed it, i made it so that printing a image waits until the terminal responds, as otherwise it could lead to a race condition, which in viu could manifest as never actually displaying the image at all.
To do this response, i had to add i=10 (image identifier), as otherwise kitty seemingly does not respond at all.

I don't understand when that happens.

As a result of using the same id of 10, only the last picture is shown. If you run `viu` a few times and scroll up, they will have been replaced.